apps/opik-backend/src/main/java/com/comet/opik/domain/ProjectMetricsDAO.java

```diff
@@ -268,27 +278,65 @@ TO parseDateTimeBestEffort(:end_time)
                 STEP <step>;
             """;
 
+    private static final String GET_THREAD_DURATION = """
+            WITH thread_duration_summary AS (
+                SELECT
+                    tt.id,
+                    min(t.start_time) as thread_start_time,
+                    max(t.end_time) as thread_end_time,
+                    if(max(t.end_time) IS NOT NULL AND min(t.start_time) IS NOT NULL
+                                AND notEquals(min(t.start_time), toDateTime64('1970-01-01 00:00:00.000', 9)),
+                            (dateDiff('microsecond', min(t.start_time), max(t.end_time)) / 1000000.0),
```

Comment:
Shouldn't it be ` 1000.0` ?  Why `1000000.0`?

Reply:
You're totally right, thanks for the catch! Pushed a fix and validated it carefully